tedana/viz.py

```diff
@@ -245,6 +245,41 @@ def write_kappa_scatter(comptable, out_dir):
     plt.close()
 
 
+def write_kappa_scree(comptable, out_dir):
+    """
+    Creates a scree plot sorted by kappa, showing the values of the kappa and
+    rho metrics as well as the variance explained.
+
+    Parameters
+    ----------
+    comptable : (C x X) :obj:`pandas.DataFrame`
+        Component metric table. One row for each component, with a column for
+        each metric. Requires at least four columns: "classification",
+        "kappa", "rho", and "variance explained".
+    out_dir : :obj:`str`
+        Figures folder within output directory
+
+    """
+
+    fig, ax1 = plt.subplots(figsize=(10, 9))
+
+    ax1.plot(comptable.index.to_numpy(), comptable['variance explained'].to_numpy(),
```

Comment:
Is `to_numpy()` necessary here? I feel like matplotlib should be able to handle pandas.Series.

Reply:
lemme see if I can dig up the examples i used for testing. I feel that I did this for a reason, but I am not certain

Reply:
It may have been a lingering effect from the testing notebook, because I was loading comp_table directly. Let see what happens!